### DISCUSSION TITLE
Modes toggle position

### DISCUSSION THREAD

**Discussion by syselement** [GitbookIO/community#517] [Feature Requests]
Hello,

The light and dark mode toggle should be immediately visible when opening a page, without requiring scrolling. However, with the new platform update, the toggle is located at the bottom of the page, which necessitates scrolling.
I believe it should be positioned in the header to ensure easy access.

Thanks in advance

**Comment by Solynor** [comment]
Hello,

I completely agree, and I imagine there must be a reason why the toggle was placed there. However, the current position is quite inconvenient because every time I reach the bottom of a page, the footer pops up and shifts everything upwards to display the button—it’s really unpleasant for the user experience.

If the toggle could simply be positioned in the header next to the search button, it would be much better and more intuitive. Alternatively, perhaps a solution could be found to prevent the footer from popping up when reaching the bottom of a page.

Thanks in advance for considering this!

**Reply by Solynor** [reply]
Thanks for the update! The new version improves the toggle's visibility and usability significantly. I appreciate the implementation, great job!
I'll consider this issue resolved.

**Reply by syselement** [reply]
Beautiful! Same here ✌️
